cmd/helm/dependency.go

```diff
@@ -71,6 +71,36 @@ the dependency charts stored locally. The path should start with a prefix of
 If the dependency chart is retrieved locally, it is not required to have the
 repository added to helm by "helm add repo". Version matching is also supported
 for this case.
+
+A repository can be defined as a git URL. The path must start with a prefix of
+"git:" followed by a valid git repository URL.
+
```

Comment:
Indenting/name:

# Chart.yaml
dependencies:
- name: nginx
  version: "master"
  repository: "git:https://github.com/helm/helm-chart.git"

Reply:
ok.  I will fix it.